script/Arcueid/Day 1/01_00_ARC01_6.txt

```diff
@@ -48,7 +48,7 @@ I hurriedly sit down on the sofa.
 [sha:a9dfe5909cac4d2ef18a662085ebb5fc056130f9]{
 -- Page 298, Offset 1909.
```

Comment:
Can't come up with a great one off the top of my head. 'Cat got your tongue' is a bit too mocking. My kotowaza dictionary suggests 'Coy as Croker's mare' but I don't know what the authors were smoking because I have never heard that in my life

Reply:
I'd say "cat got your tongue" does kinda work? She's being a bit casually teasing, so why not?